packages/ceramic-common/src/utils/doctype-utils.ts

```diff
@@ -254,7 +254,8 @@ export class DoctypeUtils {
         if (DoctypeUtils.isSignedRecord(record)) {
             const linkedBlock = await ipfs.block.get(record.link)
             return {
-                jws: record, linkedBlock,
+                jws: record,
+                linkedBlock: new Uint8Array(linkedBlock.data.buffer),
```

Comment:
D: this might break when they switch away from buffers here. We should be careful when upgrading ipfs I guess.

Reply:
https://github.com/ipld/js-ipld-block/blob/master/src/index.js

Reply:
New code ☝️ 
I guess we could add a check if data has type Uint8array? if so just use that, otherwise use the buffer.

Reply:
I agree 👍  They changed a bunch of things in `master` so there might be more places where it can break :/

Reply:
> New code ☝️
> I guess we could add a check if data has type Uint8array? if so just use that, otherwise use the buffer.

makes sense 👍